lib/validator.ex

```diff
@@ -139,10 +139,10 @@ defmodule Elixium.Validator do
     this function must return a boolean.
   """
   def valid_transaction?(transaction, pool_check) do
-    with true <- Enum.all?(transaction.inputs, & pool_check.(&1)),
-         true <- tx_addr_match?(transaction),
-         true <- tx_sigs_valid?(transaction),
-         true <- outputs_dont_exceed_inputs?(transaction) do
+    with true <- IO.inspect(Enum.all?(transaction.inputs, & pool_check.(&1))),
```

Comment:
Let's remove the IO.inspect. We should never have an IO.inspect in master

Reply:
Removing Stray IO